File: app/src/main/java/com/nelyan/ui/NotificationActivity.java
package com.nelyan.ui;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.content.Context;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.RelativeLayout;

import com.nelyan.R;
import com.nelyan.adapter.NotificationAdapter;

public class NotificationActivity extends AppCompatActivity {
    Context mContext;
    ImageView ivBack;
    NotificationAdapter notificationAdapter;
    RecyclerView recyclerview;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_notification);
        mContext=this;
        ivBack=findViewById(R.id.ivBack);
        ivBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });

             recyclerview=findViewById(R.id.recyclerview);
        notificationAdapter = new NotificationAdapter(mContext);
        recyclerview.setLayoutManager(new LinearLayoutManager(mContext));
        recyclerview.setAdapter(notificationAdapter) ;
    }
}

File: app/src/main/java/com/nelyan/ui/NotificationActivity.kt
package com.nelyan.ui

import android.content.Context
import android.os.Bundle
import android.view.View
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.nelyan.R
import com.nelyan.adapter.NotificationAdapter

class NotificationActivity : AppCompatActivity() {
    lateinit  var mContext: Context
    var ivBack: ImageView? = null
    var notificationAdapter: NotificationAdapter? = null
    var recyclerview: RecyclerView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_notification)
        mContext = this
        ivBack = findViewById(R.id.ivBack)
        ivBack!!.setOnClickListener(View.OnClickListener { finish() })
        recyclerview = findViewById(R.id.recyclerview)
        notificationAdapter = NotificationAdapter(mContext)
        recyclerview!!.setLayoutManager(LinearLayoutManager(mContext))
        recyclerview!!.setAdapter(notificationAdapter)
    }
}